### DISCUSSION TITLE
Forgot password missing from login screen?

### DISCUSSION THREAD

**Discussion by bilogic** [orchidsoftware/platform#1616] [Q&A] [answered]
Hi,

Great work! I just found this hours ago and started testing and reading the docs. A lot of good ideas and it is really RAD.
But how can a user reset their password if they have forgotten it? I can't find anything in the docs either. I assume I'm missing something since the more advanced stuff like 2FA available. Any help is most appreciated, thank you!

![image](https://user-images.githubusercontent.com/946010/111456320-d3c5fa00-8751-11eb-9700-aded9942cae0.png)

**Comment by tabuna** [comment]
Hi, this is the default. But you can extend this by adding registration, password recovery, 2fa using https://github.com/orchidsoftware/fortify

**Reply by bilogic** [reply]
Thanks for the quick response! I had a glance at https://github.com/orchidsoftware/fortify, is password recovery already there? Or are you saying I can add code for password recovery there?

**Reply by tabuna** [reply]
He already is

**Comment by bilogic** [comment, marked as answer]
Haha, ok I finally got it. Thanks @tabuna 

To use Laravel's Jetstream/Fortify auth UI:

```BASH
# composer require laravel/fortify
# composer require orchid/fortify
```

Edit `config/platform.php` 
```PHP
// disable Orchid's auth UI and use Laravel's auth UI instead
'auth' => false,
```

Refresh your page in the browser to see Laravel's auth UI.

**Reply by tabuna** [reply]
Many different packages (UI/Fortify/Jetstream/Breeze) are released over the past year. To cover all of them, we'll take it out separately.

**Reply by bilogic** [reply]
Yes, very good architecture decision!
